src/dodal/beamlines/i22.py

```diff
@@ -25,10 +25,18 @@
 from dodal.utils import BeamlinePrefix, get_beamline_name
 
 BL = get_beamline_name("i22")
+IS_LAB = BL == "p38"
+LINKAM_IS_IN_LAB = False
 PREFIX = BeamlinePrefix(BL)
 set_log_beamline(BL)
 set_utils_beamline(BL)
 
+
+VISIT_PATH: Path = Path(
+    f"/dls/${'p38' if IS_LAB else 'i22'}/data/2024/cm37271-2/bluesky"
+)
+API_URL: str = f"http://{'p38' if IS_LAB else 'i22'}-control:8088/api"
```

Comment:
Should: I think `BL` contains `p38` or `i22` string can't we use that rather than the if statement here?

Reply:
good catch

Reply:
addressed